Documentation/user-guides/alerting.md

```diff
@@ -30,9 +32,72 @@ spec:
   replicas: 3
 ```
 
-The Alertmanager instances will not be able to start up, unless a valid configuration is given. The following example configuration sends notifications against a non-existent `webhook`, allowing the Alertmanager to start up, without issuing any notifications.
+The Alertmanager instances will not be able to start up, unless a valid configuration is given. To create a configuration, AlertmanagerConfig resource or manully created secret could be used.
 For more information on configuring Alertmanager, see the Prometheus [Alerting Configuration document][alerting-config].
 
+## AlertmanagerConfig Resource
+
+The following example configuration creates an AlertmanagerConfig resource that sends notifications against a non-existent `wechat` receiver, allowing the Alertmanager to start up, without issuing any notifications.
+
+[embedmd]:# (../../example/user-guides/alerting/alertmanager-config-example.yaml)
+```yaml
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: config-example
+  labels:
+    alertmanagerConfig: example
+spec:
+  route:
+    groupBy: ['job']
+    groupWait: 30s
+    groupInterval: 5m
+    repeatInterval: 12h
+    receiver: 'wechat-example'
+  receivers:
+  - name: 'wechat-example'
+    weChatConfigs:
+    - apiURL: 'http://wechatserver:8080/'
+      corpID: 'wechat-corpid'
+      apiSecret:
+        name: 'wechat-config'
+        key: 'apiSecret'
+
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: wechat-config
+data:
+  apiSecret: d2VjaGF0LXNlY3JldAo=
+```
+
+Save the above AlertmanagerConfig in a file called `alertmanager-config.yaml` and create a resource from it using `kubectl`.
+
+```bash
+$ kubectl create -f alertmanager-config.yaml
+```
+
+Alertmanager require a `alertmanagerConfigSelector` for finding those AlertmanagerConfig resources. In the previous example, the label `alertmanagerConfig: example` is added, so Alertmanager instance should be updated, adding the `alertmanagerConfigSelector`:
```

Comment:
```suggestion
The `alertmanagerConfigSelector` field in the Alertmanager resource Spec needs to be specified so that the operator can select such AlertmanagerConfig resources. In the previous example, the label `alertmanagerConfig: example` is added, so the Alertmanager instance should be updated, adding the `alertmanagerConfigSelector`:
```